### ISSUE TITLE
Pre-work Checklist: Developer: Jack Casica

### ISSUE THREAD

**Issue by JackCasica** [hackforla/website#5865]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by JackCasica** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? 

A few months. My first boarding meeting was November 6th

**Comment by JackCasica** [comment]
Progress: I have completed a pull request for my first "good first issue". Currently waiting for review. I plan to attend a team meeting this Sunday before closing this issue.
Blockers: None
Availability: 6 hours across the week.
ETA: I expect this issue to be completed by Sunday, February 4th, 2024
Pictures (optional): N/A

**Comment by ExperimentsInHonesty** [comment]
@JackCasica Hi, I wanted to follow up with you, about your work on the website team and to encourage you to complete areas where you have not done any issues yet.  I have filled out this checklist based on the work I can see via the PR and Issue tab

- [x] Setting up your local environment from a contributing file
- [x] GitHub branching
- [x] Pull Requests
- [ ] Reviewed other people's Pull Requests
- [ ] Resolved Merge Conflicts
- [ ] Written documentation for other Developers (Architecture, etc.)
- [ ] Mentored other developers
- [ ] Making issues for other developers to work on
- [ ] API requests
- [ ] Cron Job Scripting (GitHub Actions)
- [ ] CRUD operations

### Next steps for you
- [Review two PRs from this list ](https://github.com/hackforla/website/pulls?q=is%3Apr+is%3Aopen+review%3Arequired+-label%3A%22Complexity%3A+Medium%22+-label%3A%22Ignore%3A+Test%22+-label%3ADraft)
- Do a front end Medium issue from this list [Front end issues in the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=-label%3A%22role%3A+dev+leads%22+-label%3A%22role%3A+design%22+label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257)

**Comment by JackCasica** [comment]
> Review two PRs from this list
Do a front end Medium issue from this list [Front end issues in the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=-label%3A%22role%3A+dev+leads%22+-label%3A%22role%3A+design%22+label%3A%22complexity%3A+medium%22+label%3A%22role%3A+front+end%22#column-7198257)

Hi @ExperimentsInHonesty ~ Thanks for the followup and next steps! I'll will follow through with these this week.

**Comment by JackCasica** [comment]
### Progress: "What is the current status of your project? What have you completed and what is left to do?"
I have completed all tasks except for: 
>  Attend weekly team meetings:

### Blockers: "Difficulties or errors encountered."
 Prioritizing in my schedule.
 
### Availability: "How much time will you have this week to work on this issue?"
I will attend today's office hours from 7-8PM PST.

### ETA: "When do you expect this issue to be completed?"
End of day.

### Pictures (optional): "Add any pictures of the visual changes made to the site so far."
This prework issue does not make changes to the HFLA website.

**Comment by JackCasica** [comment]
Progress: 
Currently waiting for medium frontend tasks.
Reviewing Reviewing PR Review Guide.
Planning to attend this Sunday's All Hands.

ETA: "When do you expect this issue to be completed?"
End of week

**Comment by JackCasica** [comment]
**Progress: "What is the current status of your project? What have you completed and what is left to do?"**
I have attended two dev meetings and one office hours meeting. I plan to attend next Sunday's All Hands meeting

**Blockers: "Difficulties or errors encountered."**
No blockers at the moment.

**Availability: "How much time will you have this week to work on this issue?"**
All day Sunday (April 7)

**ETA: "When do you expect this issue to be completed?"**
By next Sunday (April 7)

**Comment by JackCasica** [comment]
I have attended all weekly meetings including: 
- [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
- [x] (Optional) Office Hours, Thursdays 7-8pm PST
- [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST

This was my last blocker and I will be closing this issue now ~

**Comment by HackforLABot** [comment]
@JackCasica

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 13, 2024 at 12:14 PM PST.</sub>

**Comment by HackforLABot** [comment]
@JackCasica

Please add update using the below template (even if you have a pull request). Afterwards, remove the '2 weeks inactive' label and add the 'Status: Updated' label.
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (optional): "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 1) place your issue in the `Questions/In Review` column of the Project Board and ask for help at your next meeting, 2) put a "Status: Help Wanted" label on your issue and pull request, or 3) put up a request for assistance on the #hfla-site channel. Please note that including your questions in the issue comments- along with screenshots, if applicable- will help us to help you. [Here](https://github.com/hackforla/website/issues/1619#issuecomment-897315561) and [here](https://github.com/hackforla/website/issues/1908#issuecomment-877908152) are examples of well-formed questions.

<sub>You are receiving this comment because your last comment was before Tuesday, August 27, 2024 at 12:04 AM PST.</sub>

**Comment by HackforLABot** [comment]
The Hack for LA Bot has closed this issue due to member inactivity.

**Comment by HackforLABot** [comment]
## Activity Log: JackCasica
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- JackCasica opened: https://github.com/hackforla/website/issues/5865 at 11/06/2023, 08:01 PM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/5865 at 11/06/2023, 08:01 PM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6165 at 01/30/2024, 12:22 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6165#issuecomment-1917843375 at 01/30/2024, 12:33 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-1917847490 at 01/30/2024, 12:36 PM PST
- JackCasica opened: https://github.com/hackforla/website/pull/6198 at 01/30/2024, 12:52 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-1917899570 at 01/30/2024, 01:10 PM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6045 at 01/30/2024, 01:20 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6045#issuecomment-1917928715 at 01/30/2024, 01:31 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6045#issuecomment-1917932724 at 01/30/2024, 01:34 PM PST
- JackCasica unassigned: https://github.com/hackforla/website/issues/6045 at 01/30/2024, 01:34 PM PST
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6198#event-11679900147 at 02/01/2024, 10:46 PM PST
- JackCasica issue closed by PR 6198: https://github.com/hackforla/website/issues/6165#event-11679900281 at 02/01/2024, 10:46 PM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6048 at 02/02/2024, 09:37 AM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6048#issuecomment-1924412268 at 02/02/2024, 09:56 AM PST
- JackCasica opened: https://github.com/hackforla/website/pull/6214 at 02/02/2024, 10:58 AM PST
- JackCasica commented on: https://github.com/hackforla/website/pull/6214#issuecomment-1925031679 at 02/02/2024, 06:52 PM PST
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6214#event-11709057210 at 02/05/2024, 11:39 AM PST
- JackCasica issue closed by PR 6214: https://github.com/hackforla/website/issues/6048#event-11709057474 at 02/05/2024, 11:39 AM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6200 at 02/05/2024, 06:46 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6200#issuecomment-1928685510 at 02/05/2024, 06:48 PM PST
- JackCasica opened: https://github.com/hackforla/website/pull/6243 at 02/05/2024, 06:59 PM PST
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6243#event-11736569221 at 02/07/2024, 10:59 AM PST
- JackCasica issue closed by PR 6243: https://github.com/hackforla/website/issues/6200#event-11736569440 at 02/07/2024, 10:59 AM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6187 at 02/07/2024, 07:45 PM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/6187#issuecomment-1933318252 at 02/07/2024, 07:46 PM PST
- JackCasica opened: https://github.com/hackforla/website/pull/6263 at 02/07/2024, 08:36 PM PST
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6263#event-11755041805 at 02/09/2024, 12:16 AM PST
- JackCasica issue closed by PR 6263: https://github.com/hackforla/website/issues/6187#event-11755041955 at 02/09/2024, 12:16 AM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-1971570832 at 02/29/2024, 09:00 AM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-1971589698 at 02/29/2024, 09:10 AM PST
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-1985875081 at 03/08/2024, 07:16 AM PST
- JackCasica assigned: https://github.com/hackforla/website/issues/6490 at 03/22/2024, 08:28 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6490#issuecomment-2015522855 at 03/22/2024, 10:02 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6490#issuecomment-2015525984 at 03/22/2024, 10:03 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6522#issuecomment-2022973781 at 03/27/2024, 07:52 AM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6522#pullrequestreview-1964587657 at 03/27/2024, 12:38 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6535#issuecomment-2025402557 at 03/28/2024, 07:44 AM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6535#pullrequestreview-1967042287 at 03/28/2024, 12:02 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-2026512273 at 03/28/2024, 07:42 PM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6546#pullrequestreview-1967815491 at 03/28/2024, 09:30 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6490#issuecomment-2028833618 at 03/31/2024, 10:25 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/5865#issuecomment-2028846592 at 03/31/2024, 11:02 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6549#issuecomment-2030197676 at 04/01/2024, 10:23 AM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6549#pullrequestreview-1972036196 at 04/01/2024, 01:07 PM PDT
- JackCasica issue closed as completed: https://github.com/hackforla/website/issues/6490#event-12350206284 at 04/03/2024, 06:32 PM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6549#pullrequestreview-1986650673 at 04/08/2024, 07:41 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6549#issuecomment-2042952062 at 04/08/2024, 07:46 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6613#issuecomment-2046321084 at 04/09/2024, 06:59 PM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6613#pullrequestreview-1993241664 at 04/10/2024, 08:58 PM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6613#pullrequestreview-1993244705 at 04/10/2024, 09:02 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6155#issuecomment-2054147916 at 04/14/2024, 11:31 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6635#issuecomment-2054149276 at 04/14/2024, 11:36 AM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6635#pullrequestreview-1999726416 at 04/14/2024, 11:36 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6708#issuecomment-2068471036 at 04/21/2024, 09:34 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6708#issuecomment-2071311420 at 04/22/2024, 07:49 PM PDT
- JackCasica submitted review: https://github.com/hackforla/website/pull/6708#pullrequestreview-2016141534 at 04/22/2024, 07:49 PM PDT
- JackCasica assigned: https://github.com/hackforla/website/issues/6683 at 04/22/2024, 08:08 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6683#issuecomment-2071326568 at 04/22/2024, 08:09 PM PDT
- JackCasica opened: https://github.com/hackforla/website/pull/6712 at 04/23/2024, 01:54 PM PDT
- JackCasica assigned: https://github.com/hackforla/website/issues/6697 at 04/23/2024, 08:02 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6697#issuecomment-2073921497 at 04/23/2024, 08:03 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6717 at 04/23/2024, 08:10 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6718 at 04/23/2024, 08:14 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6719 at 04/23/2024, 08:16 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6720 at 04/23/2024, 08:17 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6721 at 04/23/2024, 08:18 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6722 at 04/23/2024, 08:20 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6723 at 04/23/2024, 08:21 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6724 at 04/23/2024, 08:22 PM PDT
- JackCasica opened: https://github.com/hackforla/website/issues/6725 at 04/23/2024, 08:23 PM PDT
- JackCasica issue closed as completed: https://github.com/hackforla/website/issues/6697#event-12603891053 at 04/25/2024, 01:11 AM PDT
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6712#event-12635284514 at 04/28/2024, 09:40 AM PDT
- JackCasica issue closed by PR 6712: https://github.com/hackforla/website/issues/6683#event-12635284552 at 04/28/2024, 09:40 AM PDT
- JackCasica assigned: https://github.com/hackforla/website/issues/6662 at 05/05/2024, 09:04 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6662#issuecomment-2094863107 at 05/05/2024, 09:08 AM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6662#issuecomment-2094866090 at 05/05/2024, 09:18 AM PDT
- JackCasica opened: https://github.com/hackforla/website/pull/6822 at 05/05/2024, 08:24 PM PDT
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6822#event-12718842673 at 05/06/2024, 11:31 AM PDT
- JackCasica issue closed by PR 6822: https://github.com/hackforla/website/issues/6662#event-12718842899 at 05/06/2024, 11:31 AM PDT
- JackCasica assigned: https://github.com/hackforla/website/issues/6849 at 05/19/2024, 06:55 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6849#issuecomment-2119532853 at 05/19/2024, 06:55 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/issues/6849#issuecomment-2121794207 at 05/20/2024, 10:48 PM PDT
- JackCasica opened: https://github.com/hackforla/website/pull/6870 at 05/20/2024, 11:01 PM PDT
- JackCasica commented on: https://github.com/hackforla/website/pull/6870#issuecomment-2133563476 at 05/27/2024, 07:08 AM PDT
- JackCasica PR was merged: https://github.com/hackforla/website/pull/6870#event-12965140343 at 05/28/2024, 07:48 PM PDT
- JackCasica issue closed by PR 6870: https://github.com/hackforla/website/issues/6849#event-12965140437 at 05/28/2024, 07:48 PM PDT
